app/customer.py

```diff
@@ -0,0 +1,49 @@
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            car: dict
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = Car(*car.values())
+        print(f"{self.name} has {self.money} dollars")
+
+    def trip_cost(
+            self,
+            shop: Shop,
+            fuel_price: float
+    ) -> float:
+        products_cost = shop.products_cost(self.product_cart).values()
```

Comment:
The `products_cost` method returns a dictionary, but you are calling `.values()` on it, which returns a view object. Ensure that this is intended, as you later sum these values. If `products_cost` is expected to return a total cost, consider modifying the method to return a single float value instead of a dictionary.